cash-register/js/ItemList.js

```diff
@@ -0,0 +1,22 @@
+// object containing all items available for sale
+var itemList = {
+  milk: new Item('milk', 2.00),
+    eggs: new Item('eggs', 3.00),
```

Comment:
@meowbot24 Style - Keep these attributes at the same level of indentation (see how `milk` and `eggs` are indented differently?)